NineChronicles.Headless.Executable/NineChronicles.Headless.Executable.csproj

```diff
@@ -10,11 +10,14 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.5.0.37" />
-    <PackageReference Include="AWSSDK.CognitoIdentity" Version="3.5.1.2" />
+    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.7.0.5" />
```

Comment:
Since ac54068afa34a02f2bd6542cbf6c17fdb3dfd5c6, this package might be useless also. Could you check it?